### ISSUE TITLE
bundled header

### ISSUE THREAD

**Issue by mpreisler** [OpenSCAP/libswid#20]
test/unit/catch.hpp is a boost header, please consider including the header from boost instead of including it in your project. Bundling is a big red flag for downstream distribution, Fedora, Debian, RHEL and others all have rules against it.

**Comment by matejak** [comment]
I don't think that it is a boost header - it is a [single-header library](https://github.com/catchorg/Catch2) and I can't find it in the latest Boost release tarball. It uses a Boost license though.
`catch.hpp` and `acutest.h` are not installed and just used when building unittests, do you think that they can really cause problems in distributions?
